components/NewsItem.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react'
 import Upvote from './Upvote'
+import Points from './Points'
 
 class NewsItem extends React.Component {
   constructor (props, context) {
     super(props, context)
     this.state = {upvoted: this.props.upvoted}
+    this.state = {count: this.props.count}
```

Comment:
Also you forgot to add `count` to propTypes. Is there a better prop name than `count`, given this component will soon have a variety of numbers passed in as props. Maybe `pointsCount`?